README.md

```diff
@@ -1,240 +1,92 @@
+2. the _Process Engine_ (PE, or just Engine): this components receives process descriptions and executes them. The PE is available for many platforms and can easily be ported to missing architectures with our platform-independent framework.
 
-For further information, just contact us: proceed@snet.tu-berlin.de
+**Install Code Execution Environment**
 
-# Coding
+To run the latest code from this PROCEED repository, you only need to have `Node.js`, `Yarn` and `Docker` installed on your system. To know the recommended versions, please [check our wiki page](https://github.com/PROCEED-Labs/proceed/wiki/Installation-and-Configuration-for-Development). (There you will also find instructions for installing additional software recommendations if you want to extend PROCEED.)
 
-This project is organized as a Mono-Repo and contains multiple software project developed by PROCEED:
+After that, you must install the platform compiler `node-gyp` globally using Yarn:
 
-- the PROCEED Engine (DPE) for Win, Linux, Mac, Android, iOS, Browser, MCUs
-- the Management System (MS) as a NextJS Web App
-- Machine Capabilities
-- supporting libraries/modules
+`yarn global add node-gyp`
 
-## Install Development Software
+**Retrieving PROCEED's Dependencies**
 
-To develop for PROCEED you need to [install yarn](https://yarnpkg.com/en/docs/install), and also install Node > v20.11.0.
-
-For further information about installation and configuration of our favorite development environment, [see the Wiki page](https://github.com/PROCEED-Labs/proceed/wiki/Installation-and-Configuration-for-Development) and [the debugging configuration](https://github.com/PROCEED-Labs/proceed/wiki/Debugging).
-
-## Install PROCEED Dependencies
-
-After cloning the repository, you need to install the PROCEED project dependencies and all third-party libraries. Run the following command in the root directory of the repo:
+After cloning/retrieving the code, you need to install the software dependencies. Run the following command in the root directory of the repo:
 
 ```
-yarn global add node-gyp
 yarn install --ignore-engines
 ```
 
 This will install all modules and their dependencies inside `node_modules/`
 
 > Note: On macOS catalina there might be an error message popping up during the install step involving node-gyp. If it says "no xcode or clt version detected" then follow the steps from [this page of the node-gyp repo](https://github.com/nodejs/node-gyp/blob/master/macOS_Catalina.md) to resolve this problem.
 
-For the Engine and the Management System there are _development_ and _build_ commands configured in the `package.json` file.
+### Starting the Development Code
 
-**Docker Installation**
-Make sure Docker is installed on your system. If not, you can download and install Docker from the official website: (https://docs.docker.com)
+> Please note that you need access to the private Environment Configurations repository, if you want to run the authentication & authorization subsystem.
```

Comment:
This is not true for the new ms, the user needs to set the correct environment variables (in `.env.local` for example). These also don't need to be our secrets, users could generate secrets for the oauth providers on their own.